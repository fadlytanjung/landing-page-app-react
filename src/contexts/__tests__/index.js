import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Context from '../';

describe('src/contexts', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Context />);
    expect(tree).toMatchSnapshot();
  });
});
