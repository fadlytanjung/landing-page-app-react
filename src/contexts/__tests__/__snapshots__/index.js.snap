// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/contexts render 1`] = `<Context.Provider />`;
